execution/executor-benchmark/src/fake_executor.rs

```diff
@@ -83,56 +98,21 @@ impl FakeExecutor {
             DbAccessUtil::get_coin_store(&sender_coin_store_key, state_view)?.unwrap()
         };
 
-        let new_account_key = DbAccessUtil::new_state_key_account(new_account_address);
-        let new_coin_store_key = DbAccessUtil::new_state_key_aptos_coin(new_account_address);
-
-        {
-            let _timer = TIMER.with_label_values(&["read_new_account"]).start_timer();
-            let new_account_already_exists =
-                DbAccessUtil::get_account(&new_account_key, state_view)?.is_some();
-            if new_account_already_exists {
-                // This is to handle the case that we re-create seed accounts when adding more
-                // accounts into an existing db. In real VM this will be an abort, I choose to
-                // return Success here, for simplicity.
-                return Ok(TransactionOutput::new(
-                    Default::default(),
-                    vec![],
-                    0,
-                    TransactionStatus::Keep(ExecutionStatus::Success),
-                ));
-            }
-        }
-        {
-            let _timer = TIMER
-                .with_label_values(&["read_new_coin_store"])
-                .start_timer();
-            assert!(DbAccessUtil::get_coin_store(&new_coin_store_key, state_view)?.is_none());
-        }
-
         // Note: numbers below may not be real. When runninng in parallel there might be conflicts.
-        sender_coin_store.coin -= initial_balance;
+        sender_coin_store.coin -= transfer_amount;
 
         let gas = 1;
         sender_coin_store.coin -= gas;
 
         sender_account.sequence_number += 1;
 
-        let new_account = Account {
-            authentication_key: new_account_address.to_vec(),
-            ..Default::default()
-        };
-
-        let new_coin_store = CoinStore {
-            coin: initial_balance,
-            ..Default::default()
-        };
+        // add total supply via aggregators?
 
-        let mut total_supply: u128 =
-            DbAccessUtil::get_value(&TOTAL_SUPPLY_STATE_KEY, state_view)?.unwrap();
-        total_supply -= gas as u128;
+        // let mut total_supply: u128 =
```

Comment:
move this to `handle_batch_account_creation_and_transfer`?

Reply:
total supply is only affected on signer, so this is the right place to add it back